SuperBuild/External_OpenIGTLink.cmake

```diff
@@ -40,6 +40,7 @@ if(NOT DEFINED OpenIGTLink_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
       -DBUILD_TESTING:BOOL=OFF
       -DBUILD_SHARED_LIBS:BOOL=ON
       -DOpenIGTLink_PROTOCOL_VERSION_2:BOOL=ON
+      -DOpenIGTLink_PROTOCOL_VERSION_3:BOOL=ON
```

Comment:
Is it intentional that both OpenIGTLink_PROTOCOL_VERSION_2 and OpenIGTLink_PROTOCOL_VERSION_3 are on?